auth/src/main/java/com/firebase/ui/auth/ResultCodes.java

```diff
@@ -0,0 +1,33 @@
+package com.firebase.ui.auth;
+
+import android.app.Activity;
+
+/**
+ * Result codes returned when using {@link AuthUI.SignInIntentBuilder#build()} with
+ * {@code startActivityForResult}.
+ */
+public final class ResultCodes {
+    private ResultCodes() {
+        // We don't want people to initialize this class
+    }
+
+    /**
+     * Sign in succeeded
+     **/
+    public static final int OK = Activity.RESULT_OK;
+
+    /**
+     * Sign in canceled by user
+     **/
+    public static final int CANCELED = Activity.RESULT_CANCELED;
+
+    /**
+     * Sign in failed due to lack of network connection
+     **/
+    public static final int NO_NETWORK = 10;
```

Comment:
I also removed the `RESULT_` bit since the class is named `ResultCodes` and it looks nicer without the extra `RESULT_`.